web-ng/src/app/components/project-list/project-list.component.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright 2021 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<div fxLayout="column">
+  <gnd-header-layout> </gnd-header-layout>
+
+  <h1 fxFlex class="recent-projects">Recent Projects</h1>
+  <div fxFlex class="container-projects">
+    <div class="container">
+      <mat-card (click)="onNewProject()" class="project-add">
```

Comment:
Hover behavior doesn't appear to be defined here. This should probably be a `mat-button` instead.

Reply:
we are using mat-card component to give it a card look. https://material.angular.io/components/card/overview

Reply:
Yeah I understood that part, but `mat-card` alone doesn't implement Material Design hover and click behaviors. You probably can combine with `mat-button` or just use it alone for the desired effect.

Reply:
mat-card does not accept mat-button as a child.

Reply:
Right, it would be a child of `mat-card-content`,

Reply:
If we use button in card, then only that button area will have hover effect, and yet clicking places inside card outside button would trigger the function. So is it okay to add on hover effect to the card?

Reply:
have added css to handle this  `cursor: pointer;`

Reply:
Sgtm.